src/Malawi_cor_labels_GEE script.js

```diff
@@ -0,0 +1,197 @@
+var malawi_binary = ee.Image("projects/ee-aasareansah/assets/malawi_binary"),
```

Comment:
this can just be `cropland_binary`

Reply:
the general goal is to make it so you only need to change one or two lines to make this work for another country